### DISCUSSION TITLE
Visual studio code URL Handler

### DISCUSSION THREAD

**Discussion by Ra-dot-netizen** [community/community#43003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, for a visual studio code project related, i had to install github, and create an account there, but the problem i have is that as soon as i created a github account i got an email that gave me github launch code, i copied the code and wrote it on my visual studio code thingy, and then now it's just glithing or loading i don't know, please help me because this is really stressing me out and im really new into all of this things, thx. 
![image](https://user-images.githubusercontent.com/121728933/210151604-a902c806-f685-44ba-8ab3-42b4c62af5c1.png)
This is what it shows me i don't really know if you see it but yeah, please help me.

**Comment by LuisContrerasGlz** [comment]
you can try looking for more information in the Visual Studio Code documentation or the GitHub extension documentation. You can also try reaching out to the Visual Studio Code or GitHub community forums for additional help and support.
you can try to check the internet connection, like for example if you are using a school network or a company network, also double check that you have the latest version of VS code and the GitHub extension installed correctly

**Comment by Ankush11903** [comment]
try reinstalling the program to fix this problem.
